crates/runner-host/src/main.rs

```diff
@@ -124,3 +130,17 @@ async fn main() -> Result<()> {
     Handle::current().spawn_blocking(|| Scheduler::<board::Board>::run()).await?;
     Ok(())
 }
+
+fn ensure_trunk() -> Result<()> {
+    let trunk_installed = Command::new("which").args(["trunk"]).spawn()?.wait()?.success();
+    if trunk_installed {
+        return Ok(());
+    }
+
+    let mut cargo_command = Command::new("cargo");
+    // Uses trunk version 0.19.3 due to the latest not compiling with the rustc version used by wasefire.
+    // When/ if the rustc version gets updated this should be updated as well.
```

Comment:
fyi: Using the wrapper script will/should take care of this.

Reply:
Per my other comment, changed to use the wrapper script.